installer/bundle/bundle_skel.sh

```diff
@@ -591,7 +591,7 @@ pkg_upd() {
     pkg_name=$2
     pkg_allowed=$3
 
-    echo "----- Upgrading package: $pkg_name ($pkg_filename) -----"
+    echo "----- Upgrading to package: $pkg_name ($pkg_filename) -----"
```

Comment:
I feel like the original message may be clearer; if the idea is to make it clear that we are upgrading to a new VERSION of the same package, we could do something like "Upgrading $pkg_name to version $pkg_filename" ?

Reply:
@hestolz changed, i please review. Looks more clear now. thanks.